app/components/provider_interface/status_box_components/recruited_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2 class="govuk-heading-l">Conditions of offer have been met</h2>
+<%= render SummaryListComponent.new(rows: rows) %>
+<%= render ProviderInterface::ConditionsComponent.new(application_choice: application_choice) %>
+
+<a href="/application/2/confirm-enrollment" role="button" draggable="false" class="govuk-button" data-module="govuk-button">
```

Comment:
This presumably needs to link to a real URL :)

Reply:
:-) We don't let providers enrol candidates yet, removing...